.travis.yml

```diff
@@ -16,8 +16,10 @@ jdk:
 #  srcclr: true 
 addons:
   sonarcloud:
-    organization: "apache" 
+    organization: apache
+
 script:
   # other script steps might be done before running the actual analysis
-  - sonar-scanner
-#  - mvn clean org.jacoco:jacoco-maven-plugin:prepare-agent install sonar:sonar
+  #- sonar-scanner
+  - mvn clean verify install sonar:sonar -Dsonar.login=$SONAR_TOKEN
```

Comment:
No need to pass `sonar.login=$SONAR_TOKEN`, SONAR_TOKEN is understood by the scanner